packages/recomponents/src/plugins/r-toast-manager/index.js

```diff
@@ -9,4 +9,5 @@ const RToastPlugin = (Vue, options = {}) => {
 
 RToast.install = RToastPlugin;
 
+export {RToastManager};
```

Comment:
file named as `r-toast-manager` i understand is as:
- if we have the default export then it will be the `r-toast-manager` so `export default RToastManager `
- if we have multiple exports then it could be some tools (child or methods) of `r-toast-manager`

Reply:
also it's the plugin, so it could be `export default RToastManagerPlugin `

Reply:
You're making a good point, the default export in `r-toast-manager/index.js` should be more descriptive, like `RToastPlugin` for example (instead of `RToast`).

However we still need to keep `RToastManager` as a separate named export because it will be used outside the Vue `RToastPlugin` as well